Shipr-Containerization/shipr-inventory/Dockerfile.consumer

```diff
@@ -0,0 +1,14 @@
+FROM python:3.9-slim
+
+WORKDIR /app
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+
+COPY . .
+
+RUN chown -R appuser:appuser /app
+
+USER appuser
+
+CMD ["python", "consumer.py"]
```

Comment:
I am not sure if this dockerfile even builds. Have you tried building this? I suspect line 10 is incorrect.

Reply:
Now dockerfile is building